### ISSUE TITLE
Render Events outside validRange

### ISSUE THREAD

**Issue by matthias-influry** [fullcalendar/fullcalendar#5230]
Hey everyone,

I want to use the Calendar to select a specific range to get to a list of available events.
This particular instance of the Calendar is only responsible for range-selection, there is another instance of the calendar that is used for event organization (including drag & drop etc).

In the `select-range` instance, the validRange object starts with today, making all past days invalid. That is necessary to prevent range-selection in the past.

But it comes with a drawback: past events are not visible. The eventRender function only renders dates within the validRange. 
Since our events are flexible in date, it would help very much to also see events outside the valid range. Because if there are dates in the past that are not "used", it would be possible to go to the event-organization instance and drag the past, un-used, events into the future. 
But since my past events are not showing, the range-selection instance does not know they exist.

So long story short: it would help us a lot if we could optionally render events outside the validRange as well :-)

**Comment by acerix** [comment]
You could use `selectAllow` instead to prevent selecting in the past but still render events.

**Comment by arshaw** [comment]
i think this existing ticket captures this need: https://github.com/fullcalendar/fullcalendar/issues/5120

**Comment by arshaw** [comment]
&uarr;
Could you please visit the link, **give a** 👍  **to vote** ([?](https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments)), press the 🔈**Subscribe** button to receive updates?

**Comment by wootwoot1234** [comment]
@matthias-influry did you ever find a solution to this?  I'm having the same issue and I'm trying to figure out a work around.
